### DISCUSSION TITLE
Great job.

### DISCUSSION THREAD

**Discussion by system05** [issalig/toshiba_air_cond#2] [Ideas]
Hi issalig,

many thanks and really great job. Communication on AB line is a specific and communication on serial line between indoor and outdoor unit is much more interesting for more applications.
Toshiba has a new communication protocol TU2C-Link for a new products Estia R32 and this type of communication will be on all other units (now on VRF system SMMSu, will be on new RAV unit and new RAS generation also).
Please if you are interested on a new research this type of communication, please let me know on mail: willow.mee(at)gmail.com
One more and again. Great job and thanks for description!
I will go testing..

W.

**Comment by issalig** [comment]
Thanks, nice to see other people are interested in the project.

I do not know these new products and it will be nice to know sth about the protocol.

**Comment by system05** [comment]
Hi issalig,
can you write me your mail Please?
Thanks.
W.

S pozdravem,

Matlocha


Odesláno z mobilního zařízení

**Comment by Andrea-IT** [comment]
Dear Issalig, I am interested in the project, May I have your e-mail to contact you, thanks a lot, A.
